beacon-chain/blockchain/head_test.go

```diff
@@ -209,8 +217,8 @@ func Test_notifyNewHeadEvent(t *testing.T) {
 			Block:                     newHeadRoot[:],
 			State:                     newHeadStateRoot[:],
 			EpochTransition:           true,
-			PreviousDutyDependentRoot: genesisRoot[:],
-			CurrentDutyDependentRoot:  make([]byte, 32),
+			PreviousDutyDependentRoot: make([]byte, 32),
```

Comment:
couldn't this still be the genesis root?is that the same as this empty one?

Reply:
I left the duties to be empty for genesis, either one would work though. The only thing that matters is when the root changes, not what the root is.